recipes/workflows/translations.mdx

```diff
@@ -0,0 +1,61 @@
+---
+title: 'Translations'
+sidebarTitle: 'Translations'
+description: 'Translate notification content to different languages using i18n'
+---
+
+Translation allows you to seamlessly adapt your workflows to different languages and automatically apply them based on your users locales. It enhances engagement, personalizes the user experience, and allows you to expand your market reach.
+
+**Use case:** Facebook is a social media with users from all over the world. It allows the user to use Facebook in their language and expect to get emails from Facebook in their selected language.
+
+In the example below uses [i18n](https://www.npmjs.com/package/i18n) in simple text email but if you use React to create your emails you can use for example [react-i18next](https://react.i18next.com/) and then just pass the variables needed for it. Before using i18n to translate, locale on the subscriber is used to set to the language that this email should be translated to.
+
+```typescript
+import { workflow } from '@novu/framework';
+import i18next from 'i18next';
+import en from './locales/en.json';
+import de from './locales/de.json';
+
+i18next.init({
+  fallbackLng: 'en',
+  resources: {
+    en: {
+      translation: en,
+    },
+    de: {
+      translation: de,
+    },
```

Comment:
@davidsoderberg 

In the Novu v1 translation, we used format `en_US`, `de_DE` and recommend the same format for `subscriber.locale` field. Shall we use this format here?

Reply:
No as it is not needed really, I actually like to use other keys here to disconnect them from each other. Maybe it should be added in this recipe that they do not relate to each other what do you think?

Reply:
As you are referencing `subscriber.locale` [here](https://github.com/novuhq/docs/pull/670/files#diff-2c229eda406d79bd407aee5aa8b92dc52fc091862d9d9bbd8cf1c8d9c37a2f4dR36). Wouldn't it break for users who are using V1 have locale already stored want to test v2.

I agree with you. We can add info about this in this recipe

Reply:
Yeah that would not work, but if we add a sentance about that this is not related to v1 I think it is okay, they can also edit the example before running it.